### ISSUE TITLE
Options cannot be properly reset to default value

### ISSUE THREAD

**Issue by rugk** [keepassxreboot/keepassxc-browser#106]
## Expected Behavior

When i press "Save" in the settings, the option should be applied.

## Current Behavior

It needs a browser restart (I think).

## Steps to Reproduce (for bugs)

1. Set "Blink Time" to 500ms.
2. Go to a website where the icon will blink, because of a new password. 
3. Trigger that.
4. It blinks 500ms.
5. Change the option back to the default. Press save.
6. Go to the other website again. (in the other tab)
7. Trigger the blink.
-> It blinks only 500ms again.

## Debug info

KeePassXC-Browser Version: 1.0.1
KeePassXC Version: 2.3.1
Operating system: Linux
Browser: Firefox 59
Proxy used: yes

**Comment by rugk** [comment]
Actually, I can't get this back to work even after a KeePassXC and browser restart… (at different times, though) Something is wrong here… :laughing:

**Comment by varjolintu** [comment]
Cannot reproduce this. Worked just fine here. Used Firefox 59 too.

**Comment by rugk** [comment]
Okay, I think I see what happened. Just looked at the settings again, and I think it misset them to `0`:

![grafik](https://user-images.githubusercontent.com/11966684/38247194-9b699744-3744-11e8-8475-5ab2bc16ed29.png)

Note that in the STR at step 5 I basically just removed all characters (numbers) in the input field, so it'll show me the the default text. Somehow, when saving this seems to get interpreted as "0". Like some JS-comparison issue… :smiley: 
Note that in the UI it still showed me an empty input field until I restarted the browser now and opened the settings again. (Could be I just restored the session before…)
